dcapal-frontend/src/components/languageSwitcher.js

```diff
@@ -1,22 +1,31 @@
 import React from "react";
 import { useTranslation } from "react-i18next";
+import {
+  Select,
+  SelectTrigger,
+  SelectContent,
+  SelectValue,
+  SelectItem,
+} from "@/components/ui/select";
 
 function LanguageSwitcher() {
   const { i18n, t } = useTranslation();
 
   return (
-    <select
-      className="pl-3 pr-1.5 py-2 border focus-visible:outline-1 rounded-md text-center border-gray-300 hover:border-gray-500 focus-visible:outline-gray-600 leading-normal"
-      aria-label="Change language"
-      value={i18n.resolvedLanguage}
-      onChange={(e) => {
-        const newLang = e.target.value;
+    <Select
+      defaultValue={i18n.resolvedLanguage}
+      onValueChange={(newLang) => {
         i18n.changeLanguage(newLang);
       }}
     >
-      <option value="en"> {t("language.en")} </option>
-      <option value="it"> {t("language.it")} </option>
-    </select>
+      <SelectTrigger className="w-32" aria-label="Change language">
```

Comment:
I'm not too convinced about the size of the language selector button, maybe we can make it smaller?

```suggestion
      <SelectTrigger
        className="h-6 min-w-[4rem] w-auto px-2"
        aria-label="Change language"
      >
```

Reply:
Agree, it was definitely too big. Chose to keep the default height and padding to keep visual consistency as we'll use more of them.